docs/handbook.md

```diff
@@ -3,6 +3,8 @@ layout: page
 title: Babel Plugin Handbook
 description: This document covers how to create Babel plugins.
 permalink: /docs/handbook/
+package_source: babel-handbook
+package: babel-handbook
```

Comment:
At least now the Github button works, "Edit this page" redirect to the README of the handbook and the package in npm is not found